src/components/AdditionalInfo/AdditionalInfo.jsx

```diff
@@ -11,6 +11,23 @@ export default class AdditionalInfo extends React.Component {
     this.expandedContentId = uniqueId('tooltip-');
     this.state = { open: false };
   }
+  componentDidMount() {
+    this.setEventListener(this.expandedContentId, 'click');
+    this.setEventListener(this.expandedContentId, 'keydown');
+  }
+
+  setEventListener = (elemId, eventHandler) => {
+    const elem = document.getElementById(`${elemId}-button`);
+    if (elem) {
+      elem.addEventListener(eventHandler, event => {
+        const key = event.key || event.keyCode;
+        if (key === '' || key === 32 || eventHandler === 'click') {
+          event.preventDefault();
+          this.toggle();
+        }
+      });
+    }
+  };
```

Comment:
This looks a bit complicated. If possible I'd rather something like `onClick={toggleHandler}` and `onKeydown={toggleHandler}` in the JSX, and then define the handler function.

Reply:
Not sure why I didn't do it this way in the first place. Think I got too deep into being focused on the codepen example trying to figure out the conditional that I stuck too close to it. Have pushed an update to this that tested well in storybook preview.